dotCMS/src/main/java/com/dotcms/workflow/helper/WorkflowHelper.java

```diff
@@ -1363,39 +1344,39 @@ private IsNewAndCloneItResult isNewAndCloneIt(final String actionId) {
 
     /**
      * Resolve the role based on the id
+     * 
      * @param id String
      * @return Role
      * @throws DotDataException
      */
     private Role resolveRole(final String id) throws DotDataException {
 
         Role role = null;
-        final String newid = id.substring
-                (id.indexOf("-") + 1, id.length());
+        final String newid = id.substring(id.indexOf("-") + 1, id.length());
```

Comment:
Codacy found an issue: [String.indexOf(char) is faster than String.indexOf(String).](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5639225)